src/bigint.rs

```diff
@@ -3250,7 +3250,10 @@ impl BigInt {
             !exponent.is_negative(),
             "negative exponentiation is not supported!"
         );
-        assert!(!modulus.is_zero(), "divide by zero!");
+        assert!(
+            !modulus.is_zero(),
+            "attempt to calculate zero modulus!"
```

Comment:
@cuviper is this message better?

Reply:
Maybe "attempt to calculate with zero modulus"?